internal/sockstate/netstat_linux.go

```diff
@@ -120,6 +122,23 @@ func parseIPv4(s string) (net.IP, error) {
 	return ip, nil
 }
 
+func parseIPv6(s string) (net.IP, error) {
```

Comment:
There's a `net.ParseIP` function already https://golang.org/pkg/net/#ParseIP, which parses both IPv4 and IPv6

Reply:
It's not an IP in the usual form the the format used by `/proc/net/tcp` like `0201A8C0:88F0`.